build.sbt

```diff
@@ -8,6 +8,7 @@ lazy val util = (project in file("util")).
     libraryDependencies ++= utilDependencies
   )
 
+
```

Comment:
remove the unintended changes so that the reviewers can focus on the main changes.